lua/terrortown/lang/en.lua

```diff
@@ -1662,12 +1663,12 @@ L.header_inventory_pickup = "Inventory Weapon Pickup"
 L.header_sprint_general = "Sprint Settings"
 L.header_playersettings_armor = "Armor System Settings"
 
-L.help_killer_dna_range = "When a player is killed by another player a DNA fingerprint is left on their body. The max range convar defines the maximum distance in hammer units for DNA samples to be left. If the killer is further away, then no sample is left at the corpse."
-L.help_killer_dna_basetime = "The basetime in seconds until a DNA sample is decayed. A factor of the squared killer distance is substracted from this basetime."
-L.help_dna_radar = "The TTT2 DNA scanner shows the exact distance and direction of the selected DNA sample if equipped. However, there is also a classic DNA scanner mode that updates the selected sample with an in-world rendering everytime the cooldown has passed."
-L.help_idle = "The idle mode is used to move idle players into a forced spectator mode. To leave this mode again, they have to disable 'enforce spectator mode' in their 'gameplay' settings."
+L.help_killer_dna_range = "When a player is killed by another player, a DNA sample is left on their body. The setting below defines the maximum distance in hammer units for DNA samples to be left. If the killer will be further than this value when the victim dies, then no sample will be left on the corpse."
+L.help_killer_dna_basetime = "The base time in seconds until a DNA sample is decayed, if the killer was 0 Hammer units away. The farther the killer, the less time will be given to the DNA sample to decay."
```

Comment:
I will not remove "base" from this line since it's imrotant to mention that it's the time used to calculate the actual time based on the base time and the killer distance. Also, it's not reduced for each unit, but rather a percentage based on the current distance and maximum distance.

Reply:
Line updated.

Reply:
> I will not remove "base" from this line since it's important to mention that it's the time used to calculate the actual time based on the base time and the killer distance.

That's fair. I just thought the second sentence is enough to described that and the 'base' felt unnecessary.

> Also, it's not reduced for each unit, but rather a percentage based on the current distance and maximum distance.

Correct, but to what detail do we want to describe it in the tooltip? i don't know to be honest. Maybe we'll just leave it like this for now (or forever).

Reply:
I see no reason to explain how the formula works for any regular server owner/developer. For any unregular one, they can find out by themself, don't they?